about.html

```diff
@@ -0,0 +1,129 @@
+        <a href="#" class="navlink" id="nav-link-tr-4">Sponsor</a> 
+        <a href="#" class="navlink" id="nav-link-tr-5">News</a>
+        <a href="#" class="navlink" id="nav-link-tr-6">DW Campaign</a>        
+      </nav>
+    </nav>
+  </header>
+  <main>
+    <section id="headlineAbout">
+      <div class="container-main cnt-abt">
+        <p class='title-tag ff-2 clr-2 al-center'>"Hello! Digital World"</p>
+        <h1 class="about-head ff-1 clr-2 al-center" id="About-title">
+          STOCK MARKET<br>GLOBAL SUMMIT 2023
+        </h1>
+        <div class="abt-desc" id="abt-main-desc">
+          <p class="sec-text ff-2 al-center" id="desc-text">
+            The need for Financial freedom, Stock Market and Trading. 
+            Here from global financial experts on the best practices
+            to achieve goal.
+          </p>
+          <div class="pattern-2" id="desc-pat-tr-2"></div>
+        </div>
+        <p class="al-center" id="duration">
+          For More Details and participation, please contact us at <br>
+          DW-summit;gmail.com
+        </p>
+      </div>             
+    </section>
+    <section class="abt-sec">
+      <div  class="container-main abt-logo cnt-abt">
+        <h2 class="head4 ff-2 al-center">
+         STOCK MARKET<br>GLOBAL SUMMIT 2023 LOGO
+        </h2>
+        <hr>
+        <img src="images/dw.jpg" alt="DW Logo" id="brand-logo">
+      </div>
+    </section>
+    <section class="abt-sec">
+      <div class="container-main abt-logo cnt-abt" id = 'past-summit'>
+        <h2 class="head4 ff-2 al-center">
+          See the past DW Global Summits
+        </h2>
+        <hr>
+        <div class="summit-cnt">
+          <div class="summits" id="summit-1">
+            <h2 class="clr-4">
+              2021
+            </h2>
+            <p class="clr-4">
+              DW Global Summit 2021 in India
+            </p>
+          </div>
+          <div class="summits" id="summit-2">
+            <h2 class="clr-4">
+              2022
+            </h2>
+            <p class="clr-4">
+              DW Global Summit 2022 in USA
+            </p>
+          </div>
+        </div>
+        
+      </div>
+    </section>
+  </main>
+  <footer class="footer bg-dark" id="abt-footer">
+    <div class="container-main mob-only" id="abt-partners-ph">
+      <h3 class="head3 clr-4 fw" id="partners">Partners</h3>
+      <hr>
+      <div class="partners-list">
+        <div class="partners">
+          <img src="images/ec.png" alt="ec">
+        </div>
+        <div class="partners">
+          <img src="images/offsec2.png" alt="offsec2">
+        </div>
+        <div class="partners">
+          <img src="images/microsoft-r-logo-png-12.png" alt="microsoft-r-logo-png-12">
+        </div>
+        <div class="partners">
+          <img src="images/cisco.png" alt="cisco">
+        </div>
+        <div class="partners">
+          <img src="images/mcAfee.png" alt="mcAfee">
+        </div>
+      </div>      
```

Comment:
**[OPTIONAL BUT NECESSARY FOR IMPROVEMENT]** Though you added the `alt` attribute to images please consider making them professional by adding values that match the image or define some detail about an image for screen readers or if the images are not shown so users can read them.

## USEFULL LINKS:-

Following are some useful links to understand them:-

[LINK1](https://blog.hubspot.com/marketing/image-alt-text)
[LINK3](https://www.w3schools.com/tags/att_img_alt.asp)
[LINK3](https://moz.com/learn/seo/alt-text)